cypress/integration/client_authorization_test.spec.ts

```diff
@@ -149,10 +149,11 @@ describe("Client authentication subtab", () => {
     authenticationTab.cancel();
   });
 
-  it("Should copy auth details", () => {
-    authenticationTab.goToExportSubTab();
+  it.only("Should copy auth details", () => {
+    authenticationTab.waitForExportLoad(() => {
+      authenticationTab.goToExportSubTab();
+    });
```

Comment:
Do we need to use a callback here? I think you can just call this method and get the same result.
```suggestion
    authenticationTab.waitForExportLoad();
    authenticationTab.goToExportSubTab();
```

Reply:
no you have to intercept the call first before you can wait